smart-contracts/contracts/hooks/CaptchaHook.sol

```diff
@@ -2,13 +2,14 @@
 pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts/utils/cryptography/MessageHashUtils.sol";
 import "@unlock-protocol/contracts/dist/PublicLock/IPublicLockV12.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
 contract CaptchaHook is Ownable {
   mapping(address => bool) public signers;
 
-  constructor() {}
+  constructor() Ownable(msg.sender) {}
```

Comment:
Maybe use `_msgSender()` from the underlying `Context` contract, just in case `msg.sender` is overridden somewhere downstream by OpenZeppelin? Also for `GuildHook.sol`.